proto-build/src/main.rs

```diff
@@ -347,7 +345,6 @@ fn compile_ibc_protos_and_services(out_dir: &Path) {
     tonic_build::configure()
         .build_client(true)
         .build_server(false)
-        .format(true)
```

Comment:
Note: this option was removed in `tonic` v0.7, which now uses the `prettyplease` crate for formatting:

https://github.com/hyperium/tonic/commit/d6c0fc112b2288a080fd0a727453b24d666e3d79

This also results in slightly different formatting from before (hence `TONIC_CLIENT_ATTRIBUTE` needing to change)